README.md

```diff
@@ -101,10 +99,12 @@ Please consider the following :
 1. Fork it!
 2. Create your feature branch (under `dev` branch)
 3. Add your functions/methods to proper files
-4. Pass all CI tests
-5. Update `CHANGELOG.md`
+4. Add standard `docstring` to your functions/methods
+5. Add tests for your functions/methods (`doctest` testcases in `test` folder)
```

Comment:
The `doctest` could be misleading in a way (it does not refer to any specific artifact in the folder or file), I think we can remove this word.

Reply:
fixed in https://github.com/sepandhaghighi/penney/pull/63/commits/b35d9808f1ce64e2ea98e611689e3d96e1de7ab3